src/Controls/samples/Controls.Sample.SingleProject/Maui.Controls.Sample.SingleProject.csproj

```diff
@@ -12,6 +12,8 @@
     <WindowsPackageType>MSIX</WindowsPackageType>
     <ApplicationTitle>.NET MAUI Controls</ApplicationTitle>
     <ApplicationId>com.microsoft.maui.sample</ApplicationId>
+    <ApplicationId Condition="$(TargetFramework.Contains('-windows'))">f9e4fa3e-3505-4742-9b2b-d1acdaff4ec8</ApplicationId>
```

Comment:
Windows uses a GUID for this, so we unfortunately need to condition. Maybe we can have an `$(ApplicationGuid)`? Then we don't need this and we can potentially use that GUID in other OS. Not sure what Tizen uses? @rookiejava